pom.xml

```diff
@@ -64,10 +64,10 @@
   </scm>
 
   <properties>
-    <revision>2.12.5</revision>
+    <revision>2.12.4-1</revision>
     <changelist>-SNAPSHOT</changelist>
     <java.level>8</java.level>
-    <jenkins.version>2.222.4</jenkins.version>
+    <jenkins.version>2.249.1</jenkins.version>
```

Comment:
https://github.com/jenkinsci/jenkins/pull/4660 was in `2.235` using an LTS that is newer than that